### PR TITLE
feat: Add Mimir Wallet

### PR DESCRIPTION
**Pull request by jarvandev**
- inject mimir sdk when app is launch
- re-build tx when sign transaction in mimir

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -102,5 +104,13 @@
     "vite-tsconfig-paths": "^4.3.2",
     "vitest": "^1.6.0"
   },
-  "packageManager": "yarn@4.1.1"
+  "packageManager": "yarn@4.1.1",
+  "resolutions": {
```
Comment (Leouarz) [inline package.json:108]: Would it be possible to not add resolutions ? Why add them in the first place ? Something we can update ?

File: src/main.tsx [hunk 2]
```diff
@@ -19,11 +19,14 @@ import 'theme/index.scss';
...
+import { injectMimirIfNecessary } from 'mimir';
 
 const rootElement = document.getElementById('root');
 if (!rootElement) {
   throw new Error('Failed to find the root element');
 }
 const root = createRoot(rootElement);
 
-root.render(<App />);
+injectMimirIfNecessary().then(() => {
```
Comment (Leouarz) [inline src/main.tsx]: Let's not have this blocking call and inject it somewhere else like an effect or else

**Review by Leouarz — CHANGES_REQUESTED** [review]
Thanks for the PR
Overall looks good to me and tests are ok, mostly the blocking call at the beginning needs to be changed then we can proceed with merging.

**Comment by jarvandev** [comment]
- We fixed the issue with blocking call
- There are multiple versions of `@polkadot/*` in the project, so we use resolutions to lock the version of `@polkadot/*`, for this problem, you can see the `@polkadot/*` in `node_modules`

File: src/main.tsx (hunk 2 above)
Comment (jarvandev) [reply src/main.tsx]: We fixed the issue with blocking call

File: package.json (hunk 1 above)
Comment (jarvandev) [reply package.json:108]: There are multiple versions of @polkadot/* in the project, so we use resolutions to lock the version of @polkadot/*, for this problem, you can see the @polkadot/* in node_modules

**Review by Leouarz — APPROVED** [review]

**Pull request merged**